### PR TITLE
Video retire and redirect only

### PR DESCRIPTION
**Pull request by lhcole613**
@pebaum The retire and redirect only portion of ADO task 5627155.

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/admin/setup/domains-faq.yml [hunk 1]
```diff
@@ -233,3 +233,19 @@ sections:
             
           Want to know more about becoming the admin for your school? [Learn all about it](/microsoft-365/education/deploy/becoming-an-admin-in-office-365-education
           ).
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/setup/domains-faq.yml]:
```suggestion
          Want to know more about becoming the admin for your school? [Learn all about it](/microsoft-365/education/deploy/becoming-an-admin-in-office-365-education).
```

File: microsoft-365/admin/setup/domains-faq.yml [hunk 2]
```diff
@@ -233,3 +233,19 @@ sections:
...
+          > [!VIDEO https://www.microsoft.com/videoplayer/embed/4839abc6-9323-4cbf-a79d-2907235f9ebb?autoplay=false]
+
+          1. Search for a web hosting provider where you can build your website.
+              You may find it easiest to choose the website builder at the same company where your domain name is registered.
+          
+          1. Design your business website, making sure to include your business logo, business purpose, text, images, and links.
+          
+          1. To integrate the website with Microsoft 365, you can add your business email to the website and a link to Microsoft Teams, SharePoint, or other collaboration tool.
+          
+          1. Make sure to connect your website to your domain name, which you&#39;re also using for email on Microsoft 365.
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/setup/domains-faq.yml:249]:
```suggestion
          1. Make sure to connect your website to your domain name that you're also using for email on Microsoft 365.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @lhcole613, kindly commit the few suggested changes. Many thanks.

**Comment by pebaum** [comment]
please do not merge this pr - we're waiting on other changes before this should go live

File: microsoft-365/admin/setup/domains-faq.yml (hunk 2 above)
Comment (lhcole613) [reply microsoft-365/admin/setup/domains-faq.yml:249]: As per @pebaum request, please don't merge this PR. Thank you.

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request closed without merging**